test/index.test.js

```diff
@@ -72,9 +109,21 @@ describe('index', () => {
         Executor = require('../index');
 
         executor = new Executor({
-            host: 'jenkins',
-            username: 'admin',
-            password: 'fakepassword'
+            ecosystem,
+            jenkins: {
+                host: 'jenkins',
+                username: 'admin',
+                password: 'fakepassword',
+                nodeLabel
+            },
+            buildScript,
+            cleanupScript,
+            cleanupWatchInterval
+        });
+        compiledJobXml = _.template(TEST_XML)({
```

Comment:
It's exactly as you say!  I have changed the test code with different xml data.